### PR TITLE
modifications for storage slot set/unset and sequential read/write/clear

### PR DESCRIPTION
**Pull request by ghost**
Part of FuelLabs/fuel-specs#395

### CODE REVIEW

File: src/instruction.rs [hunk 1]
```diff
@@ -219,6 +219,9 @@ impl From<Instruction> for u32 {
         let repr = OpcodeRepr::from_u8(parsed.op);
 
         let args = match repr {
+            OpcodeRepr::SRWQ
```
Comment (Voxelot) [inline src/instruction.rs:222]: This should probably be moved down to the last section along with the other opcodes that use all 4 registers like `MEQ`

Comment (mitch-fuel) [reply src/instruction.rs:222]: whoops, I missed that that section already existed. Will move now

Comment (mitch-fuel) [reply src/instruction.rs]: fixed

**Review by Voxelot — APPROVED** [review]
lgtm

**Pull request merged**